### ISSUE TITLE
Questions about focus management

### ISSUE THREAD

**Issue by benlesh** [WICG/html-in-canvas#14]
I'm curious about how focus/blur will be managed with this.

Will the tab index be part of the containing document? So if I'm in a button outside of the canvas, and there's a button inside of the canvas, can I tab to it?

Is there a way to tap into the focus management? Some canvas-based apps have rolled their own focus management, and will want to integrate with this new feature.

**Comment by schenney-chromium** [comment]
This may be a back compatibility issue to address of we change how the canvas grabs focus. We would not want the mere existence of the feature to break existing canvas content, even if the app never calls placeElement().

**Comment by chrishtr** [comment]
> Will the tab index be part of the containing document? So if I'm in a button outside of the canvas, and there's a button inside of the canvas, can I tab to it?

Yes. This is how it already works with content under `<canvas>`.

**Comment by progers** [comment]
The https://github.com/WICG/html-in-canvas?tab=readme-ov-file#live-demo-source-using-the-drawelementimage-api-to-draw-a-pie-chart-with-multi-line-labels example demonstrates this.
